webpack.config.js

```diff
@@ -1,31 +1,29 @@
-const path = require('path');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 module.exports = {
-   entry: './main.js',
-   devtool: 'eval-source-map',	
-   output: {
-      path: path.join(__dirname, '/bundle'),
-      filename: 'index_bundle.js'
-   },
-   devServer: {
-      inline: true,
-      port: 1337
-   },
-   module: {
-      rules: [
-         {
-            test: /\.jsx?$/,
-            exclude: /node_modules/,
-            loader: 'ts-loader',
-
-         }
-      ]
-   },
-   plugins:[
-      new HtmlWebpackPlugin({
-         template: './index.html'
-      })
-   ]
-}
-
+  entry: "./src/main.js",
+  devtool: "eval-source-map",
+  output: {
+    path: path.join(__dirname, "/bundle"),
+    filename: "index_bundle.js",
+  },
+  devServer: {
+    inline: true,
+    port: 1337,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
```

Comment:
This test won't match files with .ts extension.
You will be able to write typescript code in js files but it is quite counterintuitive.

Just a note.
No action needed.